src/Locales/en-gb/Pode.psd1

```diff
@@ -0,0 +1,285 @@
+    cacheStorageNotFoundForSetExceptionMessage                            = "Cache storage with name '{0}' not found when attempting to set cached item '{1}'"
+    methodPathAlreadyDefinedExceptionMessage                              = '[{0}] {1}: Already defined.'
+    errorLoggingAlreadyEnabledExceptionMessage                            = 'Error Logging has already been enabled.'
+    valueForUsingVariableNotFoundExceptionMessage                         = "Value for '`$using:{0}' could not be found."
+    rapidPdfDoesNotSupportOpenApi31ExceptionMessage                       = "The Document tool RapidPdf doesn't support OpenAPI 3.1"
+    oauth2ClientSecretRequiredExceptionMessage                            = 'OAuth2 requires a Client Secret when not using PKCE.'
+    invalidBase64JwtExceptionMessage                                      = 'Invalid Base64 encoded value found in JWT'
+    noSessionToCalculateDataHashExceptionMessage                          = 'No session available to calculate data hash.'
+    cacheStorageNotFoundForRemoveExceptionMessage                         = "Cache storage with name '{0}' not found when attempting to remove cached item '{1}'"
+    csrfMiddlewareNotInitializedExceptionMessage                          = 'CSRF Middleware has not been initialised.'
+    infoTitleMandatoryMessage                                             = 'info.title is mandatory.'
+    typeCanOnlyBeAssociatedWithObjectExceptionMessage                     = 'Type {0} can only be associated with an Object.'
+    userFileDoesNotExistExceptionMessage                                  = 'The user file does not exist: {0}'
+    routeParameterNeedsValidScriptblockExceptionMessage                   = 'The Route parameter needs a valid, not empty, scriptblock.'
+    nextTriggerCalculationErrorExceptionMessage                           = 'Looks like something went wrong trying to calculate the next trigger datetime: {0}'
+    cannotLockValueTypeExceptionMessage                                   = 'Cannot lock a [ValueTypes]'
+    failedToCreateOpenSslCertExceptionMessage                             = 'Failed to create OpenSSL cert: {0}'
+    jwtExpiredExceptionMessage                                            = 'The JWT has expired.'
+    openingGuiMessage                                                     = 'Opening the GUI.'
+    multiTypePropertiesRequireOpenApi31ExceptionMessage                   = 'Multi-type properties require OpenApi Version 3.1 or above.'
+    noNameForWebSocketRemoveExceptionMessage                              = 'No Name for a WebSocket to remove supplied.'
+    maxSizeInvalidExceptionMessage                                        = 'MaxSize must be 0 or greater, but got: {0}'
+    iisShutdownMessage                                                    = '(IIS Shutdown)'
+    cannotUnlockValueTypeExceptionMessage                                 = 'Cannot unlock a [ValueTypes]'
```

Comment:
This can just be `[ValueType]`, to avoid confusion with `[ValueTypes]` which doesn't exist :) unsure if this is the same in other translations.